gradio/utils.py

```diff
@@ -635,12 +635,8 @@ def function_wrapper(
 
         @functools.wraps(f)
         async def asyncgen_wrapper(*args, **kwargs):
```

Comment:
why were these lines removed? So it's just not using before_fn and after_fn anymore in generators?